static/js/public/store/filters.js

```diff
@@ -210,17 +200,114 @@ class Filters {
       e.preventDefault();
       this.removeFilter("sort");
       this.addFilter("sort", el.value);
+      this.syncSortUI("sortMobile", el.value);
+
+      this.cleanFilters();
+      this.updateHistory();
+    });
+  }
+
+  initMobileSortEvents(el) {
+    el.addEventListener("change", (e) => {
+      e.preventDefault();
+      this.removeFilter("sort");
+      this.addFilter("sort", e.target.value);
+      this.syncSortUI("sort", e.target.value);
 
       this.cleanFilters();
       this.updateHistory();
+
+      // hide the drawer once clicked
+      el.classList.remove("is-active");
+    });
+  }
+
+  initFilterEvents(el) {
+    const filters = el.querySelectorAll("input");
+    const resetButton = el.querySelector("[data-js='filter-reset']");
+    const submitButton = el.querySelector("[data-js='filter-submit']");
+
+    if (filters) {
+      filters.forEach((filterEl) => {
+        filterEl.addEventListener("click", () => {
+          const filterType = filterEl.dataset.filterType;
+          const filterValue = filterEl.value;
+
+          if (this.filterExists(filterType, filterValue)) {
+            this.removeFilter(filterType, filterValue);
+          } else {
+            this.addFilter(filterType, filterValue);
+          }
+
+          this.cleanFilters();
+          this.updateHistory();
+        });
+      });
+    }
+
+    // Show shadow when scrolling through the filters on mobile
+    const elTitle = el.querySelector("[data-js='js-shadow-title']");
```

Comment:
This looks interesting https://css-scroll-shadows.now.sh/?bgColor=ffffff&shadowColor=979797&pxSize=9
Perhaps we don't need any JS at all?